api/source/service/STIGService.js

```diff
@@ -1161,7 +1161,7 @@ exports.getCcisByRevision = async function(benchmarkId, revisionStr, userObject)
   if (revisionStr != 'latest') {
     joins = ['revision r']
 
-    let [results, version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+    let [version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
```

Comment:
regex returns what it captures positionally, so we need "results" here, even though it is not used.